### DISCUSSION TITLE
We need help with new functionality

### DISCUSSION THREAD

**Discussion by stravnik** [toxic-repos/toxic-repos#30] [Q&A] [answered]
Есть идея ввести теги для записей в базе.
Теги будут отображать принадлежность записей, например, к какому либо языку программирования.
Форма подачи следующая: имя записи в базе - теги(через запятую)

There is an idea to introduce tags for entries in the database.
Tags will display the belonging of the records, for example, to any programming language.
The submission form is as follows: record name in the database - tags (separated by commas)

**Comment by sergiomarotco** [comment, marked as answer]
Не интересно никому
